### DISCUSSION TITLE
Option to change the default output directory of generated files for `quarto use binder`

### DISCUSSION THREAD

**Discussion by chainsawriot** [quarto-dev/quarto-cli#9298] [Feature Requests]
### Description

The current default is to add the generated files (e.g. `PostBuild`, `runtime.txt`) to the root directory `.` of the project. `repo2docker`supports these files to be in `./.binder` or `./binder` ([an example](https://github.com/jupyterlab/jupyterlab-demo/tree/5b5a10dbd9fdecefb06436618f700db8a124a619/.binder) and [documentation](https://repo2docker.readthedocs.io/en/latest/specification.html)).

It would be nice if we can change the default directory from the root directory to `./.binder` or `./binder`. It separates cleanly what files are generated by quarto for binder.

**Comment by cderv** [comment]
Thanks ! I moved this to #9313 as it should be easy enough to add unless I am missing something. Less follow there.
